segmentation/include/pcl/segmentation/extract_polygonal_prism_data.h

```diff
@@ -128,6 +129,16 @@ namespace pcl
       inline void 
       setInputPlanarHull (const PointCloudConstPtr &hull) { planar_hull_ = hull; }
 
+       /** \brief Provide a vector of the concave polygons indices, as recieved from ConcaveHull::polygons.
+       * \note This is only needed when using ConcaveHull that has more than one ring.
+       * \param[in] rings - see ConcaveHull::polygons
+       */
+      inline void
+      setRings(const std::vector<pcl::Vertices>& rings)
```

Comment:
Is "rings" really the best word for this? In your example case it fits, but more generally, it could also just be two non-intersecting squares. Maybe "polygons" would be better, using the same name as in the reconstruct functions? Or "hull indices", same name as in crop_hull.h?